.github/ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "📊 Tracking Issue"
+title: "📊Tracking: [service]"
+about: Add a module tracking issue (for project management)
+labels: management/tracking
+---
+
+### Add your +1 👍  to help us prioritize high-level constructs for this service
+--- 
+
+### Overview:
+<!-- 
+Summary of the service (leverage the service’s product page for the text) and a link to the relevant AWS Docs. This should be the same text that we put at the top of the package’s README.db. Also include a link to the service’s CDK Construct Library API reference page. 
```

Comment:
1. Use a decent line wrap throughout this file, typically 120 or some use 80

2. typo: readme.md